app/src/main/java/com/fcaputo/parkingapp/utils/Constants.kt

```diff
@@ -3,4 +3,6 @@ package com.fcaputo.parkingapp.utils
 object Constants {
     const val SIZE_NOT_SET_INT = -1
     const val SIZE_NOT_SET_STRING = "-"
```

Comment:
Here, you can use a ZERO_INT and EMPTY_STRING, it's not a requirement but is more reusable.

Reply:
I separated the empty string ("") from the string I used to state size not set ("-"), for clarity purposes. I think this way it is more readable that the size is undefined, rather than just missing